addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2003,7 +2003,9 @@
     @extend %o-preview-alpha-background;
     font-family: sans-serif !important;
     font-size: 15px !important;
-    padding: $o-we-sidebar-content-field-spacing $o-we-sidebar-content-field-spacing $o-we-sidebar-content-field-spacing*.8;
+    // Include border in padding to avoid that gradient background color gets
+    // redrawn on each side.
+    padding: $o-we-item-border-width + $o-we-sidebar-content-field-spacing $o-we-item-border-width + $o-we-sidebar-content-field-spacing $o-we-item-border-width + $o-we-sidebar-content-field-spacing * .8;
```

Comment:
is this still necessary? I don't see issue with the old code

Reply:
Without this, the border of the preset preview is wrongly painted with the background is a gradient:
![image](https://github.com/odoo/odoo/assets/71644421/5cab79f3-5f6c-4564-818f-d8426cf8b4a0)